### DISCUSSION TITLE
View Transitions API support in SSE swaps

### DISCUSSION THREAD

**Discussion by drderuiter** [bigskysoftware/htmx#1976] [Ideas]
I've been using `transition:true` in `hx-swap`. Really cool and powerful. I'd love to be able to use the same in `sse-swap`.

I tried, naively, doing something similar in the SSE extension as what's currently being done [here](https://github.com/bigskysoftware/htmx/blob/953ce54b7e8210748e8f3e2eb357808cd418d450/src/htmx.js#L3624). Unsurprisingly, that did not work.

Would it be possible to add this?

**Comment by nickchomey** [comment]
I'd love this as well. Sse and view transitions are the two things I'm most interested in using through htmx
